index.html

```diff
@@ -594,52 +596,57 @@
 \begin{equation}
-\bar{\boldsymbol{\alpha}} = \frac{\boldsymbol{\alpha}} {1  - g (1 - \boldsymbol{\alpha})} \ ,
+\mathbf{C} = \frac{(1 - \mathbf{s})(1 - 0.139 \mathbf{s})}{1 + 1.17 \mathbf{s}} \ , \quad
+\mathbf{s} = \sqrt{\frac{1 - \boldsymbol{\alpha}}{1 - \boldsymbol{\alpha}\, g}} \ .
 \end{equation}
-and remapped extinction coefficient $\bar{\boldsymbol{\mu}}_t$ by
+Inversion of this gives the per-channel single-scatter albedo $\boldsymbol{\alpha}$ in terms of the **`subsurface_color`** $\mathbf{C}$ and anisotropy $g$:
 \begin{equation}
-\bar{\boldsymbol{\mu}}_t = \frac{1  - g (1 - \boldsymbol{\alpha})}{1 - g} \boldsymbol{\mu}_t \ .
+\boldsymbol{\alpha} = \frac{1 - \mathbf{s}^2}{1 - g\,\mathbf{s}^2} \ , \quad
+\mathbf{s}          = 4.09712 + 4.20863\mathbf{C} - \sqrt{9.59217 + 41.6808C + 17.7126\mathbf{C}^2} \ .
 \end{equation}
-Note that due to the singularity at $g=1$, in practice the anisotropy needs to be limited to be not too close to 1.
+This gives one approximate $\boldsymbol{\alpha}(\mathbf{C})$ mapping, but as noted it ignores the presence of the dielectric boundary. Other approximate forms which take into account the dielectric interface are presented in various sources (e.g. [#d'Eon2021], [#Burley2018]).
+We also note that the relevant approximation may depend on the specific assumptions made about the subsurface interface, for example in some cases renderers may approximate the boundary BSDF via a diffuse lobe, for efficiency and improved convergence. In that case, it would be more appropriate to use an $\boldsymbol{\alpha}(\mathbf{C})$ designed for this specific interface BSDF, in order to satisfy equation [subsurface_albedo_constraint].
+
+The phase function anisotropy $g$ of the medium is taken to be given directly by the **`subsurface_anisotropy`** parameter [^anisotropy_g]. It is assumed that the phase function has the standard Henyey-Greenstein form.
 
-Implementations may choose to compute light transport in the subsurface medium either via a fast, approximate bidirectional scattering surface reflectance distribution (BSSRDF) model using e.g. the dipole approximation [#Jensen2001], or more accurately via volumetric path tracing ([#Novak2018], [#Pharr2023]).
+Once the underlying volumetric medium properties $\boldsymbol{\mu}_t$, $\boldsymbol{\alpha}$ and $g$ are determined, implementations may choose to compute light transport in the subsurface medium either via a fast, approximate bidirectional scattering surface reflectance distribution (BSSRDF) model using e.g. the dipole approximation [#Jensen2001], or more accurately via volumetric path tracing ([#Novak2018], [#Pharr2023]).
```

Comment:
> fast, approximate BSSRDF model

This seems fine, and approximations seem useful especially in a rasterization context, although I would personally try to avoid making these approximations sound too good for two main reasons:
- The inherent inaccuracies that they involve can result in visual artifacts and extra complexity to cover up those artifacts.
- These approximations are not always faster, especially when efficient sampling is used in the path-traced version or when the path lengths are short because the albedo is low.

Specifically, I'm wondering whether we should remove the word "fast" from the description.

Reply:
> implementations may choose to compute light transport in the subsurface medium either via a ... BSSRDF model ... or ... via volumetric path tracing

There may be other less common algorithms or variations that could also be used, so I wonder whether we should add the phrase "for example" or something like that. (I'm thinking about other options like photon mapping, VPLs, photon beams, screen-space blurring, etc.)

Reply:
Suggested rewording:

>Once the underlying volumetric medium properties $\boldsymbol{\mu}_t$, $\boldsymbol{\alpha}$ and $g$ are determined, implementations are free to choose how to compute the light transport in the subsurface medium according to their use case, for example accurately via volumetric path tracing ([#Novak2018], [#Pharr2023]), or approximately via a  bidirectional scattering surface reflectance distribution (BSSRDF) model using e.g. the dipole approximation [#Jensen2001].


```suggestion
Once the underlying volumetric medium properties $\boldsymbol{\mu}_t$, $\boldsymbol{\alpha}$ and $g$ are determined, implementations are free to choose how to compute the light transport in the subsurface medium according to their use case, for example accurately via volumetric path tracing ([#Novak2018], [#Pharr2023]), or approximately via a  bidirectional scattering surface reflectance distribution (BSSRDF) model using e.g. the dipole approximation [#Jensen2001].```